Xamarin.Essentials/SecureStorage/SecureStorage.ios.cs

```diff
@@ -13,12 +13,6 @@ public static partial class SecureStorage
 
         public static Task SetAsync(string key, string value, SecAccessible accessible)
         {
-            if (string.IsNullOrWhiteSpace(key))
```

Comment:
This actually needs to remain as this is an iOS specific SetAsync that takes in a SecAccessible. The PlatformGetAsync is fine.